src/protocols/ssh/clipboard.c

```diff
@@ -47,14 +48,30 @@ int guac_ssh_clipboard_blob_handler(guac_user* user, guac_stream* stream,
     /* Append new data */
     guac_client* client = user->client;
     guac_ssh_client* ssh_client = (guac_ssh_client*) client->data;
-    guac_terminal_clipboard_append(ssh_client->term, data, length);
+    guac_common_clipboard_append(ssh_client->clipboard, (char*) data, length);
 
     return 0;
 }
 
 int guac_ssh_clipboard_end_handler(guac_user* user, guac_stream* stream) {
 
-    /* Nothing to do - clipboard is implemented within client */
+    guac_client* client = user->client;
+    guac_ssh_client* ssh_client = (guac_ssh_client*) client->data;
+
+    char output_data[GUAC_COMMON_CLIPBOARD_MAX_LENGTH];
+
+    const char* input = ssh_client->clipboard->buffer;
+    char* output = output_data;
+    guac_iconv_write* writer = ssh_client->settings->clipboard_crlf ? GUAC_WRITE_CP1252_CRLF : GUAC_WRITE_CP1252;
```

Comment:
This will result in the clipboard contents being transformed into [Windows CP-1252](https://en.wikipedia.org/wiki/Windows-1252).